model/coverage.py

```diff
@@ -66,7 +66,7 @@ def not_covered(cls, count_as_covered=None,
         """
         if not count_as_covered:
             count_as_covered = cls.DEFAULT_COUNT_AS_COVERED
-        elif isinstance(count_as_covered, basestring):
+        elif isinstance(count_as_covered, str):
```

Comment:
I'm a little worried about stuff like this because if you pass in a bytestring it won't be covered anymore. I don't anticipate that actually happening, though. Is the general attitude that bytestrings are so rare in Python 3 that it won't come up?